user_guide_src/source/incoming/controllers.rst

```diff
@@ -87,15 +87,48 @@ modify this by passing the duration (in seconds) as the first parameter:
 Validating Data
 ***************
 
+.. _controller-validatedata:
+
+$this->validateData()
+=====================
+
+.. versionadded:: 4.2.0
+
+To simplify data checking, the controller also provides the convenience method
+``validateData()``.
+
+The method accepts an array of data to validate in the first parameter, and an
+array of rules in the second parameter, and in the optional third parameter,
```

Comment:
Method ``validateData()`` has 4 parameters, my understanding of this text is that it has 3 parameters.

Reply:
You are correct!
I added.